scripts/switch_brain.py

```
@@ -0,0 +1,679 @@
+            # Verify the tunnel actually reaches Ollama
+            try:
+                probe = httpx.get(f"{tunnel_url}/api/tags", timeout=10)
+                if probe.status_code == 200:
+                    tunnel_models = [
+                        m["name"] for m in probe.json().get("models", [])
+                        if isinstance(m, dict) and m.get("name")
+                    ]
+                    print(ok(f"Tunnel verified — {len(tunnel_models)} model(s) reachable via {tunnel_url}"))
+                else:
+                    print(warn(f"Tunnel returns HTTP {probe.status_code} — may need auth bypass"))
+            except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)